.github/workflows/test-ansible.yml

```diff
@@ -49,7 +49,6 @@ jobs:
           env
           export PATH=/opt/python/3.6.7/bin:${PATH}
           sudo apt-get install python3 python3-pip
-          sudo pip3 install wheel
```

Comment:
Why can this be removed?

Reply:
Because we only added this in the PR https://github.com/operator-framework/operator-sdk/pull/4253/files#diff-b40bad6fd0ff24f0b76e97122134b4fb2df604d6b73e4ddbd88bf5229f6c9912R34. It was not used/set before that and it's not required.